packages/core/realtime-js/test/RealtimeClient.auth.resubscribe.test.ts

```diff
@@ -0,0 +1,192 @@
+import assert from 'assert'
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+import { testBuilders, EnhancedTestSetup } from './helpers/setup'
+import { utils } from './helpers/auth'
+import { CHANNEL_STATES } from '../src/lib/constants'
+
+let testSetup: EnhancedTestSetup
+
+beforeEach(() => {
+  testSetup = testBuilders.standardClient()
+})
+
+afterEach(() => {
+  testSetup.cleanup()
+  testSetup.socket.removeAllChannels()
+})
+
+describe('Issue #1904: Resubscribe with custom JWT fails', () => {
```

Comment:
Let's not mention number of issue specifically. My opinion is that it creates noise.